lib/temporal/errors.rb

```diff
@@ -57,10 +57,11 @@ class WorkflowAlreadyCompletingError < InternalError; end
   class WorkflowExecutionAlreadyStartedFailure < ApiError
     attr_reader :run_id
 
-    def initialize(message, run_id)
+    def initialize(message, run_id = nil)
```

Comment:
This is more interesting than it looks: in order for the error to deserialize correctly from await_workflow_result, it needs to have a 1-arg constructor. We don't have the run_id available when a child workflow fails to start.

Defaulting to nil seemed like a good way to solve both of those problems at once.

Reply:
Odd that they don't send back the run_id for child workflows.